### PR TITLE
Debian Tracefile

### PR DESCRIPTION
**Pull request by aconitumnapellus**
##### SUMMARY
This PR "backports" the fix for the debian tracefile from production to the git repository.

##### ISSUE TYPE
 - Bugfix Pull Request

### CODE REVIEW

File: templates/usr/local/bin/rsync_debian_mirror.j2 [hunk 1]
```diff
@@ -54,12 +54,21 @@ function writetracefile() {
...
     echo "Architectures: GUESSED:{${archlist}}" >> ${base_path}/${trace}.new
+    echo "Architectures-Configuration: $globalarchlist" >> ${base_path}/${trace}.new
+    echo "Throughput: 1Gb" >> ${base_path}/${trace}.new
     echo "Upstream-mirror: ${repo_source}" >> ${base_path}/${trace}.new
 
+    echo "Location: Bern/Bern/Switzerland/Europe" >> ${base_path}/${trace}.new
+    echo "Country: CH" >> ${base_path}/${trace}.new
+    echo "Maintainer: Adfinis AG" >> ${base_path}/${trace}.new
+    echo "${mirrorname}" >> ${base_path}/project/trace/_traces
+
```
[the remark this answers is not in the document]
Comment (aconitumnapellus) [reply templates/usr/local/bin/rsync_debian_mirror.j2]: Ive pushed a new version, do you mean like this?

[the remark this answers is not in the document]
Comment (martinwe-adfinis) [reply templates/usr/local/bin/rsync_debian_mirror.j2]: Yeah, that looks fine to me.

Note that I cannot comment on the general nature of this change (i.e. whether the additional lines added to the trace file are correct). But I trust that it works. :slightly_smiling_face:

File: templates/usr/local/bin/rsync_debian_mirror.j2 [hunk 2]
```diff
@@ -54,11 +54,22 @@ function writetracefile() {
...
-    echo "Architectures: GUESSED:{${archlist}}" >> ${base_path}/${trace}.new
-    echo "Upstream-mirror: ${repo_source}" >> ${base_path}/${trace}.new
+	# let those tabs be tabs from here...
+	cat <<- EOF >> "$base_path/$trace.new"
+	$(LC_ALL=POSIX LANG=POSIX date -u)
+	Date: $(LC_ALL=POSIX LANG=POSIX date -u -R)
+	Software: SySync v1
+	Running on Host: $mirrorname
+	Architectures: GUESSED:{$archlist}
+	Architecture-Configuration: $globalarchlist
```
Comment (martinwe-adfinis) [inline templates/usr/local/bin/rsync_debian_mirror.j2:64]: In your original change, this was `Architectures-Configuration` (note missing `s`). I missed that as well, sorry.

**Comment by martinwe-adfinis** [comment]
LGTM. Approving from the shell-scripting PoV. Can't comment on the correctness of the change as a whole.

**Review by martinwe-adfinis — APPROVED** [review]

**Pull request merged**